app/Services/ProjectService.php

```
@@ -231,17 +240,17 @@ public static function replicate($attribute)
     public function updateStage(ProjectStage $newStage)
     {
         $isTssUpdate = $this->project->marketing_stage->value === MarketingStage::AWARDED->value
-            && in_array($newStage->value, array_map(fn ($stage) => $stage->value, TssStage::cases()), true);
+            && in_array($newStage->value, array_map(fn($stage) => $stage->value, TssStage::cases()), true);
```

PHPMD (2.15.0)

Avoid using static access to class '\App\Enums\TssStage' in method 'updateStage'. (Clean Code Rules)

(StaticAccess)